lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -46,6 +46,14 @@
     description: The provider region name to connect to (e.g. AWS region for Amazon).
     required: false
     default: null
+  host_default_vnc_port_start:
+    required: false
```

Comment:
you are missing the version_add here, like in:
https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/influxdb.py#L32